src/include/sof/audio/channel_map.h

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Slawomir Blauciak <slawomir.blauciak@linux.intel.com>
+ */
+
+#ifndef __SOF_AUDIO_CHANNEL_MAP_H__
+#define __SOF_AUDIO_CHANNEL_MAP_H__
+
+#include <ipc/channel_map.h>
```

Comment:
You could just do forward declaration:
```
struct sof_ipc_channel_map;
struct sof_ipc_stream_map;
```
We wouldn't need to include the whole header.